cfme/utils/appliance/db.py

```diff
@@ -229,47 +227,6 @@ def setup(self, **kwargs):
 
         self.logger.info('DB setup complete')
 
-    def loosen_pgssl(self, with_ssl=False):
-        """Loosens postgres connections"""
-
-        self.logger.info('Loosening postgres permissions')
-
-        # Init SSH client
-        client = self.appliance.ssh_client
-
-        # set root password
-        cmd = "psql -d vmdb_production -c \"alter user {} with password '{}'\"".format(
```

Comment:
I think this needs to stay - set the postgres DB password to what's in the credentials file. This credentials file will later be used to create connections to the DB, and if its non-standard that will no longer work.

Reply:
Well the password is set when creating a db in `enable_internal` in `cfme/utils/appliance`